CHANGELOG.md

```diff
@@ -49,14 +73,12 @@
 - fix: Waves concat order [\#208](https://github.com/materializecss/materialize/pull/208) ([Smankusors](https://github.com/Smankusors))
 - fix: for iOS13 selecting wrong Dropdownitem on Touchdevices fix \#31 [\#199](https://github.com/materializecss/materialize/pull/199) ([wuda-io](https://github.com/wuda-io))
 - fix\(issue180\): handling invalid clickedIndex in carousel.js [\#187](https://github.com/materializecss/materialize/pull/187) ([LoganTann](https://github.com/LoganTann))
-- Fix for issue \#146 \(Rounding artefacts with dark table row\) [\#149](https://github.com/materializecss/materialize/pull/149) ([RamonvdW](https://github.com/RamonvdW))
```

Comment:
Hm, how can it happen that now we are missing some old entries?

Reply:
oh, I forgot to mention this... the CHANGELOG needs a little manual fixes 😅 

this one is missing because it could not find the commit on the main branch

I'm not sure how this happen though, but this PR changes has been bought by 55e4854de63c5faf2a495453d7ed5399ff4322cf now